pkg/api/validation/schema.go

```diff
@@ -253,7 +253,7 @@ func (s *SwaggerSchema) ValidateObject(obj interface{}, fieldName, typeName stri
 	if !ok && s.delegate != nil {
 		fields, mapOk := obj.(map[string]interface{})
 		if !mapOk {
-			return append(allErrs, fmt.Errorf("field %s: expected object of type map[string]interface{}, but the actual type is %T", fieldName, obj))
+			return append(allErrs, fmt.Errorf("expected object '%s' of type 'map[string]interface{}', but the actual type is '%T'", fieldName, obj))
```

Comment:
I tried to not be picky, but not sure what value those single quote characters adds. What appears between them does not have spaces and can be easily identified in the error message. ditto for the rest of the change.